initrd/bin/kexec-select-boot

```diff
@@ -57,7 +57,7 @@ if [ "$CONFIG_TPM2_TOOLS" = "y" ]; then
 			|| {
 			echo "FATAL: Hash of TPM2 primary key handle mismatch!";
 			echo "If you have not intentionally regenerated TPM2 primary key,";
-			warn "your system may have been compromised!";
+			warn "your system may have been compromised";
```

Comment:
This will look a bit funny now that `warn` prints `*** WARNING: $* ***` - maybe lines 59-60 should both be `warn` now so it doesn't look like it's interrupting in the middle of a sentence?

Reply:
Fixed in 8809588a :+1: